pipelines/tekton/aiedge-e2e/templates/credentials-s3.secret.yaml.template

```diff
@@ -8,4 +8,4 @@ metadata:
 
 stringData:
   s3-storage-config: |+
-    { "type": "s3", "access_key_id": "{{ YOUR_AWS_ACCESS_KEY }}", "secret_access_key": "{{ YOUR_AWS_SECRET_KEY }}", "endpoint_url": "{{ S3_ENDPOINT__https://example.amazonaws.com/ }}", "region": "{{ S3_REGION__us-west-1 }}" }
+    { "type": "s3", "access_key_id": "{{AWS_ACCESS}}", "secret_access_key": "{{AWS_SECRET}}", "endpoint_url": "{{S3_ENDPOINT}}", "region": "{{S3_REGION}}" }
```

Comment:
This will need a change in the shell tests, otherwise the PR check will fail.

Reply:
I made this change in this comment, tell me it is wrong because maybe there is more 

https://github.com/opendatahub-io/ai-edge/pull/188/commits/2ed4f2b2982def7295feb8ca29217f6aa35d9bf7